CASSANDRA.md

```diff
@@ -357,9 +339,9 @@ This section explores the extensibility capabilities of each library .
 
 There is no much offered by the driver to extend the Query Builder, e.g. add a missing CQL feature.
 
-**Phantom (v1.22.0)**
+**Phantom (v2.1.3)**
 
-You could extend Phantom by extending the DSL to add new features, although it might not be a straightforward process.
+You could extend Phantom by extending the DSL to add new features, and the process is very simple, as phantom relies on composition through implicit augmentation. Natively supporting new types is also simple to achieve through primitives.
```

Comment:
could you provide an example or add a link to the documentation that explains it? It sounds too vague.